.github/workflows/nri-config-generator.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.18'
+  GO_VERSION: '1.19'
```

Comment:
Could we chage setup-go action parameters to use the version from go.mod instead? The same we do in [nri-kubernetes](https://github.com/newrelic/nri-kubernetes/blob/main/.github/workflows/push_pr.yaml#L20) but using `nri-config-generator/go.mod` path.